deploy/csi-azurefile-node-windows-hostprocess.yaml

```diff
@@ -0,0 +1,141 @@
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-azurefile-node-win
+  namespace: kube-system
+spec:
+  updateStrategy:
+    rollingUpdate:
+      maxUnavailable: 1
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: csi-azurefile-node-win
+  template:
+    metadata:
+      labels:
+        app: csi-azurefile-node-win
+    spec:
+      serviceAccountName: csi-azurefile-node-sa
+      tolerations:
+        - key: "node.kubernetes.io/os"
+          operator: "Exists"
+          effect: "NoSchedule"
+      nodeSelector:
+        kubernetes.io/os: windows
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: type
+                    operator: NotIn
+                    values:
+                      - virtual-kubelet
+      priorityClassName: system-node-critical
+      securityContext:
+        windowsOptions:
+          hostProcess: true
+          runAsUserName: "NT AUTHORITY\\SYSTEM"
+      hostNetwork: true
+      initContainers:
```

Comment:
By the way, I don't think you need to use a full blown initContainer. You can just add a hostPath volume of type DirectoryOrCreate pointing to `/var/lib/kubelet/plugins/file.csi.azure.com`. This should prompt K8s to create the directory for you.